AdvLoggerPkg/Include/AdvancedLoggerInternal.h

```diff
@@ -46,24 +46,25 @@
 #pragma pack (push, 1)
 
 typedef volatile struct {
-  UINT32                  Signature;              // Signature 'ALOG'
-  UINT16                  Version;                // Current Version
-  UINT16                  Reserved;               // Reserved for future
-  EFI_PHYSICAL_ADDRESS    LogBuffer;              // Fixed pointer to start of log
-  EFI_PHYSICAL_ADDRESS    LogCurrent;             // Where to store next log entry.
-  UINT32                  DiscardedSize;          // Number of bytes of messages missed
-  UINT32                  LogBufferSize;          // Size of allocated buffer
-  BOOLEAN                 InPermanentRAM;         // Log in permanent RAM
-  BOOLEAN                 AtRuntime;              // After ExitBootServices
-  BOOLEAN                 GoneVirtual;            // After VirtualAddressChange
-  BOOLEAN                 HdwPortInitialized;     // HdwPort initialized
-  BOOLEAN                 HdwPortDisabled;        // HdwPort is Disabled
-  BOOLEAN                 Reserved2[3];           //
-  UINT64                  TimerFrequency;         // Ticks per second for log timing
-  UINT64                  TicksAtTime;            // Ticks when Time Acquired
-  EFI_TIME                Time;                   // Uefi Time Field
-  UINT32                  HwPrintLevel;           // Logging level to be printed at hw port
-  UINT32                  Reserved3;              //
+  UINT32      Signature;                          // Signature 'ALOG'
+  UINT16      Version;                            // Current Version
+  UINT16      Reserved[3];                        // Reserved for future

```

Comment:
Why reserving 3 here?

Reply:
The EFI_PHYSICAL_ADDRESSES were UINT64s, I have changed them to UINT32s for the offsets. So, there was an additional 4 bytes of Reserved space, so I lumped it in with the UINT16 Reserved that it was adjacent to.